lib/utils/job-utils/HTTP-tool.js

```diff
@@ -0,0 +1,195 @@
+//Copyright 2016, EMC, Inc.
```

Comment:
As a conversion, the filename is all lower case, so change filename from `HTTP-tool.js` to `http-tool.js`